src/chttpd_db.erl

```diff
@@ -407,7 +407,16 @@ db_req(#httpd{method='POST',path_parts=[_,<<"_bulk_docs">>], user_ctx=Ctx}=Req,
         true  -> [all_or_nothing|Options];
         _ -> Options
         end,
-        case fabric:update_docs(Db, Docs, Options2) of
+        case fabric_att_handler:external_store() of
+            true ->
+                couch_log:debug("store attachment externaly", []),
+                NewDocs = fabric_att_handler:att_store(Db,
+                                                          fabric:docs(Docs));
+            false ->
+                couch_log:debug("externalize attachment: disabled",[]),
+                NewDocs = Docs
+        end,
```

Comment:
It would be cleaner if the result of the case statement was assigned to `NewDocs`. i.e, `NewDocs = case...`